test/e2e/apimachinery/webhook.go

```diff
@@ -1151,6 +1154,15 @@ func registerValidatingWebhookForCRD(f *framework.Framework, context *certContex
 						Resources:   []string{"customresourcedefinitions"},
 					},
 				}},
+				NamespaceSelector: &metav1.LabelSelector{
```

Comment:
I am also adding a label selector on the deny-crd.k8s.io webhook so it won't accidentally deny crds created by other e2e tests run in parallel

Reply:
:+1: